app/src/main/res/layout/add_reminder.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The name of this layout, and some of the others, should end with "fragment.xml".

Reply:
There an issue with trying to refactor the layout name. Every time we try to change the name, it reverts back to the original title.